openedx/adg/lms/applications/tests/test_models.py

```diff
@@ -0,0 +1,82 @@
+"""
+Tests for all the models in applications app.
+"""
+
+import pytest
+
+from openedx.adg.lms.applications.models import ApplicationHub
+from student.tests.factories import UserFactory
+from .constants import USERNAME, EMAIL, PASSWORD
+
+
+@pytest.mark.django_db
+@pytest.fixture()
+def user_for_testing():
+    """
+    Create a test user
+
+    Returns:
+        User object
+
+    """
+    user = UserFactory(username=USERNAME, email=EMAIL, password=PASSWORD)
```

Comment:
If we are using these constants just to create the user then we can avoid and just create by calling UserFactory().

Reply:
This will add username, email and password defined in factory. If we don't need specific data for user we can avoid this.